src/Helpers/SearchableFinder.php

```diff
@@ -32,18 +32,17 @@ final class SearchableFinder
     private static $declaredClasses;
 
     /**
-     * @var \Illuminate\Foundation\Application
+     * @var \Illuminate\Container\Container
      */
     private $app;
 
     /**
      * SearchableModelsFinder constructor.
      *
-     * @param \Illuminate\Foundation\Application $app
-     *
+     * @param \Illuminate\Container\Container $app
```

Comment:
Maybe here should be `\Illuminate\Foundation\Application|\Laravel\Lumen\Application`.